setup.py

```diff
@@ -109,24 +75,26 @@ def read_requirements():
 s_ = setup(
     name='neat',
     version=pversion,
-    scripts=['neat/channels/compilechannels'],
+    scripts=['neat/channels/compilechannels',
+             'neat/tools/simtools/neuron/initneuron'],
     packages=['neat',
               'neat.trees',
               'neat.tools',
               'neat.tools.fittools',
               'neat.tools.plottools',
               'neat.tools.simtools.neuron',
-              'neat.tools.simtools.neuron.mech',
               'neat.channels',
               'neat.channels.channelcollection'],
+    package_data={
+        "neat.tools.simtools.neuron": ["mech/*.mod"],
+    },
     ext_package='neat',
     ext_modules=[ext],
     cmdclass={
-        'develop': PostDevelopCommand,
-        'install': PostInstallCommand,
+        'develop': DevelopCommand,
```

Comment:
(see above)

Reply:
This is the only way I got the toolbox properly in my install folder. Don't know what the problem is but at least if this one works for pip and setup.py and linux and mac we should keep it for now, even though it is ugly.

Reply:
hmm, this is really strange. the "new" classes just do what would anyway be done. ok, let's leave it for now, maybe create an issue to remind us?